.github/workflows/run-traits-tests.yml

```diff
@@ -15,6 +15,19 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
         python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        # Python 3.8 and 3.9 are no longer available on macos-latest, so
```

Comment:
I'm wondering about taking a slightly different approach here: just including `macos-13` in the matrix and then excluding just the two cases that we know don't work. That would mean that for more recent versions of Python, we'd be testing on both Intel _and_ Apple Silicon-based Macs.

Reply:
That makes sense if we're not too concerned by any additional cost that the new `macos-13` runners would introduce.